e2e/template_e2e_tests/run_e2e_test.sh

```diff
@@ -77,6 +79,20 @@ done
 export OUTPUT_FOLDER=".test_output"
 export ADMIN_EMAIL=$(gcloud auth list --filter=status:ACTIVE --format='value(account)')
 
+init_env_vars() {
+  export ADMIN_EMAIL="your_email@example.com"
+  export REGION=us-central1
+  export API_DOMAIN=localhost
+  export BASE_DIR=$(pwd)
+  export TF_VAR_project_id=${PROJECT_ID}
+  export TF_VAR_api_domain=${API_DOMAIN}
+  export TF_VAR_web_app_domain=${API_DOMAIN}
+  export TF_VAR_admin_email=${ADMIN_EMAIL}
+  export TF_BUCKET_NAME="${PROJECT_ID}-tfstate"
+  export TF_BUCKET_LOCATION="us"
+  gcloud config set project $PROJECT_ID --quiet
+}
+
```

Comment:
Could we reuse `init_env_vars.sh` from setup folder here.
```
# Set up environment variables.
setup_env_vars() {
  source "$BASE_DIR"/setup/init_env_vars.sh
}
```

Reply:
It will mix up the usage of two different projects: solutions-template-e2etest and solutions-template-develop. I added comment to explain this.